src/Random/LCG.purs

```diff
@@ -72,11 +72,13 @@ lcgM :: Int
 lcgM = 2147483647
 
 -- | Perturb a seed value
-lcgPerturb :: Number -> Seed -> Seed
+-- | Note that `Int` operations are truncated to 32-bits, so we convert to
```

Comment:
This is an implementation detail, and so it probably shouldn't be in a doc-comment.